db/revert/mutations/submit_application@1.0.0.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ccbc:mutations/submit_application from pg
```

Comment:
Where is the revert created by `sqitch rework`? There should be a revert that resets the function to its v1.0.0 behaviour